### ISSUE TITLE
Test suite compilation failure with Cabal 2.0/GHC 8.2

### ISSUE THREAD

**Issue by snoyberg** [mrkkrp/path-io#29]
```
[1 of 1] Compiling Path.IO          ( Path/IO.hs, dist/build/Path/IO.o )
Preprocessing test suite 'tests' for path-io-1.3.2..
Building test suite 'tests' for path-io-1.3.2..

tests/Main.hs:44:0: error:
     error: missing binary operator before token "("
     #if MIN_VERSION_directory(1,2,3)
     ^
   |
44 | #if MIN_VERSION_directory(1,2,3)
   | ^

tests/Main.hs:180:0: error:
     error: missing binary operator before token "("
     #if MIN_VERSION_directory(1,2,3)
     ^
    |
180 | #if MIN_VERSION_directory(1,2,3)
    | ^
`gcc' failed in phase `C pre-processor'. (Exit code: 1)
```

**Comment by mrkkrp** [comment]
Hmm, this doesn't show up on CI:

https://travis-ci.org/mrkkrp/path-io/jobs/252829902

A bug in Cabal 2?

**Comment by snoyberg** [comment]
I think you'll need to add `directory` to the test suite dependency.

**Comment by mrkkrp** [comment]
Indeed, let's try that.

**Comment by mrkkrp** [comment]
Released 1.3.3.
